t/lib/Common.pm

```diff
@@ -62,8 +65,15 @@ sub skip_unless_has_secret {
     plan skip_all => 'PERL_STRIPE_TEST_API_KEY is required' unless api_key();
 }
 
+memoize 'stripe';
 sub stripe {
     my %params = @_;
+    if ($ENV{LWP_RECORD} or !api_key()) {
+        $params{ua} = Test::LWP::Recorder->new({
+            record    => $ENV{LWP_RECORD},
+            cache_dir => 't/LWPCache/' . basename($0),
+        });
+    }
```

Comment:
@danschmidt5189 your comment got swallowed probably because I removed the `no_mock` variable. Please see the updated code.